### DISCUSSION TITLE
Allow binding volume buttons on Android devices

### DISCUSSION THREAD

**Discussion by Feodor0090** [ppy/osu#13250] [Ideas] [answered]
For now, on mobiles you mostly have to use otg/bluetooth keyboards, because most rulesets don't provide sensor controls. I noticed that osu supports binding device's side auxillary buttons if they exist (for example - google assistant button on Nokia 2.2 - it can be unbinded from GA in system settings and then binded in osu to any action), but volume buttons are ignored even if they were unbinded from system volume control through special utilities. Would be nice to make them bindable too - it will make some rulesets those need 2/3 keys (taiko, catch, rush!, tau, etc.) partially playable without external keyboard.

**Comment by peppy** [comment, marked as answer]
volume buttons would never bind to anything but volume. we are not about to allow users to break UX of their hardware buttons in a way the system would (on iOS) or should (everywhere) not allow.
